src/plugins/vis_default_editor/public/components/controls/field.tsx

```diff
@@ -69,6 +69,32 @@ function FieldParamEditor({
     );
   }
 
+  if (value && value.type === 'missing') {
```

Comment:
Can this missing being added as KBN_FIELD_TYPES in order to use this instead?

Reply:
done